app/models/concerns/audit_extensions.rb

```diff
@@ -95,8 +95,8 @@ def fix_auditable_type
     # STI Host class should use the stub module instead of Host::Base
     self.auditable_type = "Host"          if auditable_type =~  /Host::/
     self.associated_type = "Host"         if associated_type =~ /Host::/
-    self.auditable_type = auditable.type  if auditable_type == "Taxonomy" && auditable
-    self.associated_type = auditable.type if auditable_type == "Taxonomy" && auditable
+    self.auditable_type = auditable.type if auditable && ["Taxonomy", "LookupKey"].include?(auditable_type)
```

Comment:
`&& auditable` was already in the code, it's a good suggestion to refactor it but could lead to a lot of changes in unrelated tests and might affect taxonomy as well so I'd rather have it in a separate pr.

I'll move it to the end to avoid calling auditable again for anything but taxonomy and lookup keys.